src/block-components/icon/font-awesome-icon.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import {
+	faAPILoaded, faIsAPILoaded, faGetSVGIcon, faIconLoaded,
+} from '~stackable/util'
+import { pick } from 'lodash'
+
+/**
+ * WordPress dependencies
+ */
+import {
+	useState, useEffect, RawHTML, useMemo, renderToString,
+} from '@wordpress/element'
+import { Spinner } from '@wordpress/components'
+
+const FontAwesomeIcon = props => {
+	const [ forceUpdateCount, setForceUpdateCount ] = useState( 0 )
+	const forceUpdate = () => {
+		setForceUpdateCount( forceUpdateCount + 1 )
+	}
+
+	const prependRender = useMemo( () => props.prependRender ? renderToString( props.prependRender ) : '', [ props.prependRender ] )
+
+	// Wait for the FA API to load.
+	useEffect( () => {
+		faAPILoaded().then( forceUpdate )
+	}, [] )
+
+	const propsToPass = pick( props, [ 'className', 'color', 'fill', 'style' ] )
+
+	// If given an svg, just display it.
+	if ( typeof props.value === 'string' && props.value.match( /^<svg/ ) ) {
+		return <RawHTML { ...propsToPass }>{ prependRender + props.value }</RawHTML>
+	}
+
+	if ( ! faIsAPILoaded() ) {
```

Comment:
I believe this file was referenced from `src/components/font-awesome-icon/index.js`, put back some of the code comments which are not present here:

```suggestion
	// There's a chance that the Font Awesome library hasn't loaded yet, wait for it.
	if ( ! faIsAPILoaded() ) {
```